### ISSUE TITLE
Duplicate labels generated for links that are close to each other

### ISSUE THREAD

**Issue by h0tk3y** [philc/vimium#3444]
**Describe the bug**
Labels for links on a page are sometimes duplicated for different links. Trying to go to one of the two links fails, as the suffix after the common prefix of the two links is empty.

**To Reproduce**
Happened to me several times on different pages, can't find a stable way to reproduce.
This time, it was https://habr.com/ru/sandbox/start/, but, as the content on the right side is dynamic, next time the page structure may be different.

Here's a screen recording of the issue: https://drive.google.com/file/d/1nLHhHCdr0NYTM-P6UTFk03e52NSJAIu1/view?usp=sharing

**Browser and Vimium version**
Chrome Version 78.0.3904.108 (Official Build) (64-bit)
Vimium Version 1.64.6

**Comment by h0tk3y** [comment]
This might have been caused by an accidentally duplicated character in _Characters used
for link hints_ in the extension options.

**Comment by gdh1995** [comment]
Yes I think it should blamed on duplicated character in the option. BTW, since in the image there're both <kbd>p</kbd> and <kbd>sp</kbd>, does the option value also include the <kbd>Space</kbd> character?

**Comment by nerton** [comment]
Shouldn't "Characters used for link hints" filter out duplicate characters or warn of them?
